docs/src/udp.rst

```diff
@@ -6,6 +6,10 @@
 
 UDP handles encapsulate UDP communication for both clients and servers.
 
+.. note::
+    In version 1.35.0 :man:`recvmmsg(2)` and :man:`sendmmsg(2)` support was added
+    (on supported platforms). The use of these features requires a buffer larger
+    than 2 * 64KB to be passed when reading / writing data.
```

Comment:
How is this related to "writing data" at all? Does it mean that "`sendmmsg` is used if a buffer larger than 2 * 64KB is passed to `uv_udp_send()` or `uv_udp_try_send()`? How does it work? Both APIs for sending can be called with N buffers. Obviously the app cannot send UDP **datagrams** bigger than 64KB (in most cases no bigger than 1500KB), so... 

May be this means that `sendmmsg` is used if N buffers are given to `uv_udp_send()` or `uv_udp_try_send()` and their **total** len is > 2 * 64KB?

Reply:
I'll double check the code there.

Reply:
Any news or ETA? :)